policies/templates/gcp_gke_reduce_node_sa_scope_v1.yaml

```diff
@@ -0,0 +1,107 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Check the OAuth scopes on the node SA accounts.
+
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-gke-reduce-node-sa-scope
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPGKEReduceNodeSAConstraintV1
+        plural: GCPGKEReduceNodeSAConstraintsV1
+      validation:
+        openAPIV3Schema:
+          properties:
+            # The list of OAuth scopes that can be assigned to the node service account.
```

Comment:
let's use the description field instead of comment.
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#schema-object

Reply:
can we also mention the behavior when this parameter is missing?

Reply:
updated.